scripts/sandbox/generate.ts

```diff
@@ -104,27 +99,13 @@ const addStorybook = async ({
   try {
     await copy(beforeDir, tmpDir);
 
-    const packageManager = JsPackageManagerFactory.getPackageManager({ force: 'yarn1' }, tmpDir);
```

Comment:
This was the line, the CI got stuck on. 

Since it's used in a very limited way, it's not adding a lot to actually use it, so by replacing this with a direct in-file implementation the CI was no longer blocked.